### ISSUE TITLE
Error: Cannot find module './dataType'

### ISSUE THREAD

**Issue by DaiZack** [hygull/node-pandas#4]
First Run get, the same on Local, RunKit and Rel.it 
Error: Cannot find module './dataType'
![image](https://user-images.githubusercontent.com/45176371/72566092-39360680-3881-11ea-932c-615d56ed362f.png)

**Comment by shakirmengrani** [comment]
Goto node_modules/node-pandas/src/utils and change filename datatype.js to dataType.js.

**Comment by hygull** [comment]
> First Run get, the same on Local, RunKit and Rel.it
> Error: Cannot find module './dataType'
> ![image](https://user-images.githubusercontent.com/45176371/72566092-39360680-3881-11ea-932c-615d56ed362f.png)

Yeah, thank you. Actually, I have already fixed it. Just follow the above suggestion. This is known bug. In future, I am going to release a new version with more features and excellent way of accessing the columns, rows etc.
